net/apache/Makefile

```diff
@@ -8,21 +8,21 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=apache
-PKG_VERSION:=2.4.28
-PKG_RELEASE:=2
+PKG_VERSION:=2.4.34
+PKG_RELEASE:=1
 PKG_SOURCE_NAME:=httpd
 PKG_MAINTAINER:=Thomas Heil <heil@terminal-consulting.de>
 PKG_LICENSE:=Apache License
 
 PKG_SOURCE:=$(PKG_SOURCE_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=@APACHE/httpd/
-PKG_HASH:=c1197a3a62a4ab5c584ab89b249af38cf28b4adee9c0106b62999fd29f920666
+PKG_HASH:=fa53c95631febb08a9de41fd2864cfff815cf62d9306723ab0d4b8d7aa1638f0
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_NAME)-$(PKG_VERSION)
```

Comment:
PKG_BUILD_DIR is unnecessary?

Reply:
eeehhhh. I could remove it but i'd be replacing it in two places. the package is also not mine.

And unfortunately there's a really annoying build error that I haven't figured out yet.

Reply:
Ahh, it adds version.... Yeah, it should be kept.
There are several reports of mod_proxy being borked btw